### ISSUE TITLE
Tasker list on main page should have more entries

### ISSUE THREAD

**Issue by HolgerJeromin** [mapillary/mapillary_issues#2842]
The [Tasker list from the main app](https://www.mapillary.com/app/task?menu=true&focus=map&lat=20&lng=0&z=1.5) does have 5 open verifier entries

- Verify Canadian 40km/h speed limit signs
- Verify Dutch on priority road
- Verify Dutch end of priority road signs
- Verify Dutch give way signs

Clicking on a task gives a link to https://help.mapillary.com/hc/en-us/articles/115001964171
which has this list:

- No turn left in the United States
- No turn right in the United States
- No U-turn in the United States
- Speed limit 35 MPH in the United States
- Speed limit 50 MPH in the United States
- Speed limit 75 MPH in the United States
- Speed limit 30 km/h in Denmark
- Speed limit 50 km/h in Denmark
- Speed limit 70 km/h in Denmark
- Speed limit 110 km/h in Denmark
- Speed limit 20km/h in Germany
- Speed limit 60km/h in Germany
- Speed limit 100km/h in Germany
- Speed limit 120km/h in Germany
- Regulatory: End Of Buses Only
- Regulatory: Keep Right
- Regulatory: No Bicycles
- Regulatory: No Stopping
- Regulatory: Turn Right
- Warning: Falling Rocks Or Debris Left
- Warning: Loose Road Surface
- Warning: Roadworks
- Warning: Traffic Signals
- Warning: Two Way Traffic
- Warning: Uneven Road

Why are these not shown on the main map list?

**Comment by Alessandro-Barbieri** [comment]
Because the task in your are like a challenge and time limited.

**Comment by sandrauddback** [comment]
@HolgerJeromin : This is a good question! For the first version of the Tasker we have decided to include verification tasks that are requested by the community only, are local (country specific) and have a time limit as @Alessandro-Barbieri points out. The verification tasks we have listed on the help pages are tasks requested by Mapillary. I do think you have a point in which verification tasks we show where and we should consider adding all verification tasks to the Tasker. Thanks for the input!
